fleet/lib/htcondor/overlays/manke/htcondor-worker-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: htcondor-worker
+  labels:
+    app: htcondor-worker
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: htcondor-worker
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: kube-system/htcondor
+      labels:
+        app: htcondor-worker
+    spec:
+      containers:
+      - name: htcondor-worker
+        image: docker.io/lsstit/htcondor-worker:23
+        imagePullPolicy: ifNotPresent
+        env:
+        - name: CONDOR_HOST
+          value: 139.229.152.211
```

Comment:
Is `CONDOR_HOST` the `htcondor-central-manager` pod?  If so, it would be better to keep the traffic internal to k8s instead of sending it out of the cluster to and back to LB IP.  E.g. `htcondor-central-manager.htcondor`

Reply:
totally right, thanks.